descriptors/ModuleDescriptor-template.json

```diff
@@ -1,7 +1,12 @@
 {
   "id": "@artifactId@-@version@",
   "name": "Remote storage API module",
-  "requires": [],
+  "requires": [
+    {
+      "id": "inventory",
+      "version": "10.7"
+    }
+  ],
   "provides": [
     {
```

Comment:
Missing interface description and permissionSets.

Reply:
Added